frontend/spec/controllers/spree/orders_controller_ability_spec.rb

```diff
@@ -79,20 +74,17 @@ module Spree
       let(:order) { create(:order, :number => 'R123') }
 
       context '#show' do
-        context 'when token parameter present' do
-          it 'always ooverride existing token when passing a new one' do
-            cookies.signed[:guest_token] = "soo wrong"
-            spree_get :show, { :id => 'R123', :token => order.guest_token }
-            expect(cookies.signed[:guest_token]).to eq(order.guest_token)
-          end
+        context 'when guest_token correct' do
+          before { cookies.signed[:guest_token] = order.guest_token }
 
-          it 'should store as guest_token in session' do
-            spree_get :show, {:id => 'R123', :token => order.guest_token }
-            expect(cookies.signed[:guest_token]).to eq(order.guest_token)
+          it 'displays the page' do
+            expect(controller).to receive(:authorize!).with(:edit, order, order.guest_token)
+            spree_get :show, { :id => 'R123' }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.